vars/onWorker.groovy

```diff
@@ -62,10 +62,12 @@ def call(def label, def timeoutString, Closure body) {
                // To export BOTO_CONFIG, for some reason, worker did not
                // source the .profile or .bashrc anymore.
                withEnv(["BOTO_CONFIG=/home/ubuntu/.boto",
-                        "PATH=/usr/local/google_appengine:" +
+                        "PATH=" +
+                        "/home/ubuntu/webapp-workspace/devtools/khan-linter/bin:" +
+                        "/var/lib/jenkins/repositories/khan-linter/bin" +
+                        "/usr/local/google_appengine:" +
                         "/home/ubuntu/google-cloud-sdk/bin:" +
-                        "/usr/local/bin/:" +
-                        "${env.HOME}/git-bigfile/bin:" +
```

Comment:
same comment here about git-bigfile. Also, less sure that we don't use `/usr/local/bin`.